src/Microsoft.Tye.Core/ConfigModel/ConfigFactory.cs

```diff
@@ -97,5 +98,8 @@ private static ConfigApplication FromYaml(FileInfo file)
             using var parser = new YamlParser(file);
             return parser.ParseConfigApplication();
         }
+
+        private static string Normalization(string name)
```

Comment:
Can we replace with `-` instead of just removing the character? The common case is for names to look like: `MyCompany.MyDepartment.MyApp` so that would become `mycompanymydepartmentmyapp` instead of the more readable `mycompany-mydepartment-myapp`.

Reply:
Sure will do